esp/src/src-react/components/WorkunitSummary.tsx

```diff
@@ -216,13 +216,13 @@ export const WorkunitSummary: React.FunctionComponent<WorkunitSummaryProps> = ({
                             "owner": { label: nlsHPCC.Owner, type: "string", value: workunit?.Owner, readonly: true },
                             "jobname": { label: nlsHPCC.JobName, type: "string", value: jobname },
                             "description": { label: nlsHPCC.Description, type: "string", value: description },
-                            "potentialSavings": { label: nlsHPCC.PotentialSavings, type: "string", value: `${formatCost(potentialSavings)} (${Math.round((potentialSavings / totalCosts) * 10000) / 100}%)`, readonly: true },
+                            "potentialSavings": { label: nlsHPCC.PotentialSavings, type: "string", value: `${formatCost(potentialSavings)} (${totalCosts > 0 ? Math.round((potentialSavings / totalCosts) * 10000) / 100 : 0}%)`, readonly: true },
                             "compileCost": { label: nlsHPCC.CompileCost, type: "string", value: `${formatCost(workunit?.CompileCost)}`, readonly: true },
                             "executeCost": { label: nlsHPCC.ExecuteCost, type: "string", value: `${formatCost(workunit?.ExecuteCost)}`, readonly: true },
                             "fileAccessCost": { label: nlsHPCC.FileAccessCost, type: "string", value: `${formatCost(workunit?.FileAccessCost)}`, readonly: true },
                             "protected": { label: nlsHPCC.Protected, type: "checkbox", value: _protected },
                             "cluster": { label: nlsHPCC.Cluster, type: "string", value: workunit?.Cluster, readonly: true },
-                            "totalClusterTime": { label: nlsHPCC.TotalClusterTime, type: "string", value: workunit?.TotalClusterTime, readonly: true },
+                            "totalClusterTime": { label: nlsHPCC.TotalClusterTime, type: "string", value: workunit?.TotalClusterTime ? workunit?.TotalClusterTime : "0.00", readonly: true },
```

Comment:
Minor:  The following would be cleaner (but won't convert 0 => 0.00):
` workunit?.TotalClusterTime ?? "0.00"`